neqo-crypto/src/agent.rs

```diff
@@ -867,6 +867,15 @@ impl ZeroRttChecker for AllowZeroRtt {
     }
 }
 
+#[derive(Debug)]
+pub struct DisallowZeroRtt {}
+impl ZeroRttChecker for DisallowZeroRtt {
```

Comment:
This doesn't need to be public.  There is no situation, except for testing, where you would enable 0-RTT but then reject it.